README.md

```diff
@@ -107,25 +140,11 @@ If the default_lang is 'en', and the query is set to 'os', the above URL will be
 
 	https://wovn.io/contact?os=mac
 
-### 2.4. default_lang
-
-This sets the Ruby application's default language. The default value is English ('en').
-
-If a requested page, where the default language's parameter is included in the URL, the request is redirected before translating. The default_lang parameter is used for this purpose.
-
-If the default_lang is set to 'en', when receiving a request for the following URL,
-
-	https://wovn.io/en/contact
-
-The library will redirect to the following URL.
-
-	https://wovn.io/contact
-
-### 2.5 ignore_class
+### 2.6. ignore_class
 
 This sets "Ignore class" which prevent WOVN translating HTML elements that have one of the array.
```

Comment:
...which prevents WOVN from translating... (2 corrections)

Also "one of the array" is incomplete.  This sentence can't be interpreted as is.

Reply:
done, thanks.